doc/content/getting-started/troubleshooting/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Troubleshooting"
+description: ""
+---
+
+This section provides help for common issues you may encounter when getting started with {{% tts %}}.
+
+<!--more-->
+
+## My gateway won't connect. What do I do?
+
+- Double check your gateway settings in {{% tts %}}. Ensure your Gateway EUI is correct
+- Double check your configuration settings on your gateway. Is the network address the same address you use to connect to {{% tts %}}? Are your ports correct?
+- If using CUPS, double check your API Keys and certificates
+
+## My device won't join. What do I do?
+
+- Double check your Device EUI, Join EUI, Keys, and Frequency Plan settings
```

Comment:
Check frequency plan settings both for the end device and your gateways, make sure they are matching.

Reply:
Also, I think it is useful to mention LoRaWAN version (which the vendor should provide) and specifically mention `AppKey` (and possibly `NwkKey` for LoRaWAN 1.1) instead of simpley "Keys" here.